cmd/wavefront-proxy/proxy.go

```diff
@@ -174,11 +175,30 @@ func initAgent(agentID, serverURL string, service api.WavefrontAPI) {
 	agent.InitAgent()
 }
 
+func buildVersion(v string) int64 {
+	version := 0
+	s := strings.Split(v, ".")
+	for i := 0; i < len(s); i++ {
+		version *= 1e3
+		intVersion, _ := strconv.Atoi(s[i])
+		version += intVersion
+	}
+	if len(s) == 2 {
+		version *= 1e3
+	} else {
```

Comment:
Just make sure if the input is an integer, maybe change it to "else if len(s) == 1"